sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/tests/Azure.Monitor.OpenTelemetry.LiveMetrics.Demo/Program.cs

```diff
@@ -97,5 +108,37 @@ private static void GenerateTelemetry()
             // Release the allocated memory
             memoryChunk = null;
         }
+
+        private static async Task RunCPUStressTest()
+        {
+            // Get the number of available processors
+            int numProcessors = Environment.ProcessorCount;
+            numProcessors = numProcessors > 1 ? numProcessors - 1 : numProcessors;
+
+            Task[] tasks = new Task[numProcessors];
+
+            // Start each task
+            for (int i = 0; i < numProcessors; i++)
+            {
+                tasks[i] = Task.Run(() => Compute());
+            }
+
+            await Task.WhenAll(tasks);
+        }
+
+        private static void Compute()
+        {
+            var timeStamp = DateTime.Now.AddSeconds(5);
+
+            while (DateTime.Now < timeStamp)
```

Comment:
```while (DateTime.Now < timeStamp)
{}
```
Will this be any diff. interms on cpu utilization?

Reply:
> will this be any diff.

Nope. :)
I'll make the change.

![image](https://github.com/Azure/azure-sdk-for-net/assets/28785781/197bc615-1b56-4606-8094-e329a88bcb89)